google/cloud/storage/_helpers.py

```diff
@@ -187,6 +188,9 @@ def reload(
         :type if_metageneration_not_match: long
         :param if_metageneration_not_match: (Optional) Make the operation conditional on whether the
                                             blob's current metageneration does not match the given value.
+
+        :type retry: google.api_core.retry.Retry
+        :param retry: (Optional) How to retry the RPC.
```

Comment:
In order for users to use this, they'll need an explanation of what kinds of objects to pass in and how to modify them. "See retry.py" might be enough if retry.py's comments and docstrings are comprehensive.